sandpack-react/src/components/CodeEditor/CodeMirror.stories.tsx

```diff
@@ -95,6 +99,46 @@ export const Markdown: React.FC = () => (
   </SandpackProvider>
 );
 
+export const CustomLanguageShell: React.FC = () => (
+  <SandpackProvider
+    additionalLanguages={[
+      {
+        name: "shell",
+        extensions: ["sh"],
+        language: new LanguageSupport(StreamLanguage.define(shell)),
+      },
+    ]}
+  >
+    <CodeEditor
+      code={mocks.shell}
+      filePath="example.sh"
+      id="shell"
+      initMode="immediate"
+      showLineNumbers={false}
+    />
+  </SandpackProvider>
+);
+
+export const CustomLanguagePython: React.FC = () => (
+  <SandpackProvider
+    additionalLanguages={[
```

Comment:
I think you can pass this prop directly to CodeEditor instead of SandpackProvider because this kind of capability doesn't need to be shared between other components, it only matters for the CodeMirror instance.

Reply:
Oh yes this definitely simplified things 👌